checks/fixtures/zoo_test.cfg

```diff
@@ -0,0 +1,23 @@
+maxClientCnxns=200
```

Comment:
Maybe this could be in a subdirectory named "test"? I wouldn't know just looking at it what a "fixture" was.